docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -457,23 +900,25 @@ public class PersonEndpoint {
     @Transactional
     public Person findByNameForUpdate(String name){
         Person p = Person.find("name", name).withLock(LockModeType.PESSIMISTIC_WRITE).findOne();
-        //do something useful, the lock will be released when the transaction ends.
+        //Perform some action. The lock will be released when the transaction ends.
         return person;
     }
 
 }
 ----
 
-Be careful that locks are released when the transaction ends, so the method that invokes the lock query must be annotated with the `@Transactional` annotation.
+NOTE: Locks are released when the transaction ends, so the method that invokes the lock query must be annotated with the `@Transactional` annotation.
 
 == Custom IDs
 
-IDs are often a touchy subject, and not everyone's up for letting them handled by the framework, once again we
-have you covered.
+One of the advantages of Panache is that IDs are set automatically in the `PanacheEntityBase` and  `PanacheEntityRepositoryBase` classes. You can specify your own ID strategy by extending the `PanacheEntityBase` or `PanacheEntityRepositoryBase` class, depending on the entity method you are using.
```

Comment:
On the one hand, I feel like this sucked the fun out of the original sentence to make it look more producty, and I regret it. But on the other hand, I'm not sure we can justify using a COMM/ENT conditional just for tone everywhere :(

Reply:
I know what you are saying, and hey, everyone likes to have fun with language especially if it keeps people entertained and engaged. However, for product doc we have to be consistent, use parallel structure, not use slang or idioms, and be very careful with humor because it can be misinterpreted. "not everyone's up for" and "we have you covered" would probably trigger questions from the translators and some non-native speakers reading it in English might have trouble understanding.

Reply:
Yeah, I understand, and I don't see any easy way to avoid that.